test/Microsoft.AspNet.Mvc.Core.Test/ContentResultTest.cs

```diff
@@ -158,15 +118,15 @@ public class ContentResultTest
                     {
                         null,
                         "abcd",
-                        "application/xml; charset=us-ascii",
-                        "application/xml; charset=us-ascii",
+                        "text/bar; p1=p1-value",
+                        "text/bar; p1=p1-value",
                         new byte[] { 97, 98, 99, 100 }
                     },
                     {
                         null,
                         "abcd",
-                        "Invalid content type",
-                        "Invalid content type",
+                        "application/xml; p1=p1-value; charset=us-ascii",
+                        "application/xml; p1=p1-value; charset=us-ascii",
```

Comment:
good catch...this should be a valid test now as I reverted the behavior from the previous commit